### DISCUSSION TITLE
helmsman finding packages but not updating them

### DISCUSSION THREAD

**Discussion by ekhaydarov** [renovatebot/renovate#12326] [Request Help] [answered]
Thanks a lot for implementing this. I have only just got round to testing this. I can see in `lib/manager/helmsman/__fixtures__/validHelmsfile.yaml` prometheus being a valid extraction test.
So in my repo test i have prometheus but it finds the package but i get no updates for the `kube-prometheus-stack` where version should be updated to `19.1.0`
```
{
"depName": "prometheus-community-stack",
"currentValue": "19.0.1",
"lookupName": "kube-prometheus-stack",
"registryUrls": [
"https://prometheus-community.github.io/helm-charts"
],
"depIndex": 0,
"updates": []
}
```
the config file is very simple
```
{
  $schema: "https://docs.renovatebot.com/renovate-schema.json",
  vulnerabilityAlerts: {
    labels: ["security"],
    assignees: ["@ekhaydarov"]
  },
  "helmsman": {
    "fileMatch": ["(^|/)desired-state\\.yaml$"]
  },
}
```
what am i missing here?

Thanks

**Comment by rarkins** [comment]
@secustor

**Reply by secustor** [reply]
This is a bug. Fixed in https://github.com/renovatebot/renovate/pull/12330

**Comment by secustor** [comment, marked as answer]
Fixed in https://github.com/renovatebot/renovate/pull/12330
